.github/workflows/library.yml

```diff
@@ -29,7 +29,7 @@ jobs:
               run: npm run test
 
     release:
-        if: github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/next'
```

Comment:
I don't think we want to release next version. Maybe under `-next` for internal testing. What was your idea here?